resources/graphql/crdc-datahub.graphql

```diff
@@ -176,7 +186,9 @@ type Query {
         orderBy: String = "updatedAt",
         sortDirection: String = "DESC" # ["DESC", "ASC"]
     ): ListBatches
-    getBatch(batchID: ID!): Batch
```

Comment:
Is there a reason to remove getBatch API?

Reply:
Could caused during merge code.  It's should be resovled when merge 2.0.0 to this branch.

Reply:
This removing of getBatch is still in the PR.